CriticalMass/TweetTableViewCell.swift

```diff
@@ -62,8 +62,13 @@ class TweetTableViewCell: UITableViewCell, MessageConfigurable, IBConstructable
     @IBOutlet private var nameAndHandleAndTimeStackView: UIStackView!
     @IBOutlet private var nameAndHandleStackView: UIStackView!
 
+    // We need both setup methods to maintain conformance to the protocol + be able to inject a custom date for testing
     func setup(for tweet: Tweet) {
-        dateLabel.text = FormatDisplay.dateString(for: tweet)
+        setup(for: tweet, currentDate: Date())
+    }
+
+    func setup(for tweet: Tweet, currentDate: Date) {
+        dateLabel.text = FormatDisplay.dateString(for: tweet, currentDate: currentDate)
```

Comment:
If we would have ViewModels I'd say we should actually move the String creation to that one but we could at least move it from the View to the Model and make it a computed property. Also you could use the TimeTraveler for the date generation

Reply:
Also something for a future PR